_posts/2024-05-07-mai.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: April 2024
+author: Bestem0r
+---
+
+April månad er over, og vårstemninga har lagt seg i Trondheimslufta 🌼. Saman med strålande sol, grøne skogar og fuglesong, er også eksamensperioden i full gong 😔. Likevel kan det vere godt med pausar frå lesinga, og kva er vel betre enn å lese lese ei rykande fersk utgåve av Utviklarbloggen? 🤩 Finn fram godstolen, og les om kva Webkom har jobba med i april 2024! 🚀
+
+<br>
+## Utviklarbloggen har fått nytt namn
+
+Ut med det gamle, og inn med det nye! 🎉 Utviklerbloggen har fått nytt namn, og heiter frå no av **Utviklarbloggen**! 🤩 Dette er ein stor milepæl for bloggen, og eit viktig steg i riktig retning for Abakus sin språkprofil. Me håper at det nye namnet fell i smak, og at de framleis vil følgje med på kva Webkom driv med! 🥳
+
+![Utviklarbloggen](/images/posts/2024-05-07-new-name.png)
+
+<br>
+## Nye tips i interesseskjema
+
+For verksemder interesserte i Abakus, er det ikkje alltid like enkelt å vite kva ein "Bedriftspresentasjon" eller "Fukostføredrag" eigentleg dreier seg om. Me har derfor lagt til nokre **verktøytips** i interesseskjemaet, som forklarer kva dei ulike arrangementa går ut på. Dette gjer det enklare for verksemder å velje kva type arrangement dei ynskjer å arrangere! 👨‍🏫
```

Comment:
```suggestion
For verksemder interesserte i Abakus, er det ikkje alltid like enkelt å vite kva ein "Bedriftspresentasjon" eller "Frokostforedrag" eigentleg dreier seg om. Me har derfor lagt til nokre **verktøytips** i interesseskjemaet, som forklarer kva dei ulike arrangementa går ut på. Dette gjer det enklare for verksemder å velje kva type arrangement dei ynskjer å arrangere! 👨‍🏫
```

Reply:
https://ordbokene.no/bm,nn/frukost

Reply:
Or should it be the "official" name since it's an Abakus concept?

Reply:
https://ordbokene.no/bm,nn/fukost